okapi-common/src/main/java/org/folio/okapi/common/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.okapi.common.logging;
+
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
+ * The variables are stored in the local vert.x
+ * {@link io.vertx.core.Context#putLocal Context}
+ * and can be used in log4j log lines by using
+ * {@link org.apache.logging.log4j.core.lookup.StrLookup StrLookup}
```

Comment:
I don't see StrLookup in log4j2.properties. I only see references like `$${FolioLoggingContext:requestid}`. Can you make the usage explanation more instructive?

Reply:
done

Reply:
done